.github/workflows/issue1_test.yml

```diff
@@ -52,5 +55,17 @@ jobs:
           message:  ${{ steps.success.outputs.comment }}
           branch: feature1_new_title
         if: steps.test.outputs.result == '1'
+      - run: node ./.github/misc/answers.js 1 > answer.txt
+      - run: echo ::set-output name=answer::$(< answer.txt)
+        id: answer
+        shell: bash
+      - name: PR Comment Success
+        uses: Janealter/branch-pr-comment@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          message:  ${{ steps.answer.outputs.answer }}
+          branch: feature3_character_traits
```

Comment:
You need to add the correct branch here to post the comment into the appropriate PR